x-pack/metricbeat/module/openai/usage/_meta/fields.yml

```diff
@@ -0,0 +1,233 @@
+        General usage data metrics
+      fields:
+        - name: organization_id
+          type: keyword
+          description: Organization identifier
+        - name: organization_name
+          type: keyword
+          description: Organization name
+        - name: aggregation_timestamp
+          type: date
+          description: Timestamp of data aggregation
+        - name: n_requests
+          type: long
+          description: Number of requests made
+        - name: operation
+          type: keyword
+          description: Operation type
+        - name: snapshot_id
+          type: keyword
+          description: Snapshot identifier
+        - name: n_context_tokens_total
+          type: long
+          description: Total number of context tokens used
+        - name: n_generated_tokens_total
+          type: long
+          description: Total number of generated tokens
+        - name: n_cached_context_tokens_total
+          type: long
+          description: Total number of cached context tokens
+        - name: email
+          type: keyword
+          description: User email
+        - name: api_key_id
+          type: keyword
+          description: API key identifier
+        - name: api_key_name
+          type: keyword
+          description: API key name
+        - name: api_key_redacted
+          type: keyword
+          description: Redacted API key
+        - name: api_key_type
+          type: keyword
+          description: Type of API key
+        - name: project_id
+          type: keyword
+          description: Project identifier
+        - name: project_name
+          type: keyword
+          description: Project name
+        - name: request_type
+          type: keyword
+          description: Type of request
+
+    - name: dalle
+      type: group
+      description: >
+        DALL-E API usage metrics
+      fields:
+        - name: timestamp
+          type: date
+          description: Timestamp of request
+        - name: num_images
+          type: long
+          description: Number of images generated
+        - name: num_requests
+          type: long
+          description: Number of requests
+        - name: image_size
+          type: keyword
+          description: Size of generated images
+        - name: operation
+          type: keyword
+          description: Operation type
+        - name: user_id
+          type: keyword
+          description: User identifier
+        - name: organization_id
+          type: keyword
+          description: Organization identifier
+        - name: api_key_id
+          type: keyword
+          description: API key identifier
+        - name: api_key_name
+          type: keyword
+          description: API key name
+        - name: api_key_redacted
+          type: keyword
+          description: Redacted API key
+        - name: api_key_type
+          type: keyword
+          description: Type of API key
+        - name: organization_name
```

Comment:
The organizatiion_name, project_id, model_id etc have been added under data group.
We should remove them from the model specific group ?

Reply:
yeah, done!